### PR TITLE
Refactor/final

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Refactors all files

Where should your reviewer start?
- N/A

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- Everything 🥳

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: src/Hydration.js [hunk 1]
```diff
@@ -19,45 +16,38 @@ class Hydration {
...
     let lastIndexNum = [waterData.length - 1]
     let indexOfBigData = (waterData[lastIndexNum]);
-    // console.log(indexOfBigData.numOunces);
     return indexOfBigData.numOunces;
   }
 
   obtainOuncesPerDayOverAWeek(waterData) {
     let indexNum = [waterData.length - 1]
     let objectAtIndex = (waterData[indexNum]);
-    // console.log('31', objectAtIndex.date);
```
Comment (catlynbowles) [inline src/Hydration.js:36]: Thanks for cleaning up all these console logs, Irmak! 🐳

**Review by ASands17 — COMMENTED** [review]
GREAT work on these refactors! I think you did an especially great job with the css refactoring. It looks ✨  ✨  ✨  amazing ✨  ✨  ✨  !

File: test/Hydration-test.js [hunk 2]
```diff
@@ -1,315 +1,219 @@
...
         "userID": 2,
-        "date": "2019/06/21",
-        "hoursSlept": 6.7,
-        "sleepQuality": 2.9
+        "date": "2019/10/21",
+        "numOunces": 14
       },
       {
         "userID": 2,
-        "date": "2019/06/22",
```
Comment (catlynbowles) [inline test/Hydration-test.js:107]: Sorry for the hydration mix up. Thanks for fixing this! 🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡🤡

File: src/css/styles.css [hunk 3]
```diff
@@ -5,32 +5,37 @@
 body, html {
   height: 100vh;
   width: 100vw;
```
Comment (catlynbowles) [inline src/css/styles.css:7]: I like the simplicity of the styles. 💀

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**